core/injector/application_injector.cpp

```diff
@@ -925,18 +915,19 @@ namespace {
         di::bind<clock::SystemClock>.template to<clock::SystemClockImpl>(),
         di::bind<clock::SteadyClock>.template to<clock::SteadyClockImpl>(),
         di::bind<clock::Timer>.template to<clock::BasicWaitableTimer>(),
+        di::bind<clock::Ticker>.to([](const auto &injector) {
```

Comment:
I guess it is better to inject ticker without any settings and setup in in place (in ctor of target object). It will give us ability to use several different tickers in future.

Reply:
changed it a bit to get duration from babe_config
it may be injected with different args as well, let's leave it for now
changes require logic for setting and checking interval